src/Microsoft.AspNet.Authentication.OpenIdConnect/Events/OpenIdConnectAuthenticationEvents.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.AspNet.Authentication.OpenIdConnect
 {
     /// <summary>
-    /// Specifies events which the <see cref="OpenIdConnectAuthenticationMiddleware" />invokes to enable developer control over the authentication process.
+    /// Specifies events which the <see cref="OpenIdConnectMiddleware" />invokes to enable developer control over the authentication process.
     /// </summary>
     public class OpenIdConnectAuthenticationEvents : IOpenIdConnectAuthenticationEvents
```

Comment:
Good catch yeah I missed events in OAuth and OIDC, nested folders that were collapsed